web/src/lib/components/shared-components/gallery-viewer/gallery-viewer.svelte

```diff
@@ -110,7 +110,8 @@
 {#if isShowAssetViewer}
 	<AssetViewer
 		asset={selectedAsset}
-		publicSharedKey={key}
+		publicSharedKey={sharedLink?.key}
```

Comment:
do we need to pass the whole shared link and the shared link key as two separate props?